test/functional/shared.js

```diff
@@ -75,6 +75,17 @@ function makeCleanupFn(client) {
   };
 }
 
+function withTempDb(dbName, client, operation, errorHandler) {
+  return withClient(
+    client,
+    client => done => {
+      const db = client.db(dbName);
+      operation.call(this, db)(() => db.dropDatabase(done));
```

Comment:
I imagine the reason you ran into this in the first place is because the database wasn't created with a wc: majority. We might want to ensure the same here